addons/website/static/src/js/website.tour.rte.js

```diff
@@ -61,12 +61,12 @@
             },
             {
                 waitFor:   '#wrapwrap > main > div > section .row > div:first',
-                element:   '.note-popover button[data-event="color"]',
+                element:   '.note-air-popover button[data-event="color"]',
```

Comment:
Unauthorized to change website_rte
The rte is not supposed to have a different behavior for this task.